### ISSUE TITLE
yaml header has no effect

### ISSUE THREAD

**Issue by Hoffenbar** [mb21/panwriter#46]
putting this in front of a document doesn't alter anything;

```
---
title: logwrapper
style: |
  @page {
    size: A4;
    margin-top: 2cm;
  }
  h1 {
    color: purple;
  }
  p {
    font-size: 1.1em;
  }
---
```

I'd expected at least a changed h1-Header color....

**Comment by mb21** [comment]
The `style` attribute was removed in PanWriter 1.7.0. You can do this easier now by clicking the icon in the top left and adding your CSS to the "Include CSS" field, see the screenshot in https://github.com/mb21/panwriter#export-preview-to-pdf

**Comment by mb21** [comment]
Okay, I've just released PanWrtier 0.7.2 with a few bug fixes for the paged preview...
